doc/classes/AnimationNode.xml

```diff
@@ -57,15 +64,37 @@
 			</description>
 		</method>
 		<method name="_process" qualifiers="virtual const">
-			<return type="float" />
+			<return type="Dictionary" />
```

Comment:
I wonder about the cost of returning a Dictionary for a callback that's likely called frequently (every frame?).

Especially for C#, there's been a lot of debate recently around APIs that use Dictionary and force allocations.

CC @godotengine/dotnet @reduz

Reply:
Ideally, we would like to wait for the implementation of A, but I am not sure how compatible it would be with C++ structs.

Since NodeTimeInfo does the remaining time calculation in the struct, we need to allow the struct array to have a method, so when we write GDScriptStruct in C++, do we need to have _bind_method()? Well that is a bit off-topic here.

For now, this virtual function is used quite rarely, but if there are performance issues with Dictionaly, I assume it could be make it a packed Array as a temporary until GDScriptStruct is available..